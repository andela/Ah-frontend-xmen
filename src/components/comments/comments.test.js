import { shallow } from 'enzyme';
import React from 'react';
import CommentBanner from './commentBanner';
import CommentsInput from './commentInput';

const props = {
  comments: [{
    id: 1,
    body: 'a body',
    author: {
      image: '',
      username: 'dee',
    },
  }],
  user: 'dee',

};
describe('<CommentBanner/>', () => {
  it('should render without crushing', () => {
    const wrapper = shallow(<CommentBanner {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('<CommentInput/>', () => {
  it('should render with out crushing', () => {
    const wrapper = shallow(<CommentsInput />);
    expect(wrapper).toMatchSnapshot();
  });
});
