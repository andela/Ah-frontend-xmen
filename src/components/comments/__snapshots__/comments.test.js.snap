// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentBanner/> should render without crushing 1`] = `ShallowWrapper {}`;

exports[`<CommentInput/> should render with out crushing 1`] = `ShallowWrapper {}`;
